snap/snapcraft.yaml

```diff
@@ -98,7 +98,9 @@ parts:
 
         echo "Compiling pyc files..."
         # This is the last step, let's now compile all our pyc files.
-        "$SNAPCRAFT_PART_INSTALL/usr/bin/python3" -m compileall -q .
+        "$SNAPCRAFT_PART_INSTALL/usr/bin/python3" -m compileall -b -q .
+        find . -type f -name '*.py' -exec rm {} \;
+        find . -type f -name "*.so" -exec strip -s {} \;
```

Comment:
We do plan to introduce a feature flag to enable stripping in a more generic manner in the not-so-distant future, I'd perhaps wait on that for this type of change?

Reply:
This is project's (Snapcraft) decision, if it wants to have the size benefit now or later.